web/modules/custom/dpl_event/dpl_event.module

```diff
@@ -310,3 +311,36 @@ function dpl_event_preprocess_eventinstance__list_teaser(array &$variables): voi
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Altering the form for EDITING (not creating) and event series.
+ */
+function dpl_event_form_eventseries_default_edit_form_alter(array &$form, FormStateInterface $form_state): void {
+  // When saving a series, and changing the dates, the recurring_events module
+  // will make a "custom" submit button, showing a 'are you sure' confirm.
+  // However, this does not include a 'cancel' button. This is because cancel
+  // basically just means navigating away, but, this is confusing for editors.
+  // We'll add our own "button", that is really just a link to the content
+  // overview page.
+  if (!empty($form['diff']['confirm'])) {
+    $form['diff']['cancel'] = [
+      '#type' => 'link',
+      '#title' => t('Cancel all changes'),
+      // Example of a Url object from a route. See the documentation
+      // for more methods that may help generate a Url object.
+      '#url' => Url::fromRoute('system.admin_content'),
```

Comment:
Please reconsider this comment.

I do not understand how it is relevant. Every piece of code is an example of how to do something where we could reference the documentation for other examples of doing the same thing.